### PR TITLE
[Issue-116] Upgrade Pravega 0.10 to the latest snapshot version

### LINKED ISSUE #116: Upgrade Pravega 0.10 to the latest snapshot version
**Problem description**

Upgrade Pravega 0.10 to the latest commit

**Problem location**

`gradle.properties`

Suggestions for an improvement

### PR DESCRIPTION
**Pull request by thekingofcity**
Signed-off-by: zhongle.wang <zhongle_wang@dell.com>

**Change log description**

Upgrade pravega version to the latest commit pravega/pravega@a621c55

**Purpose of the change**

Fix #116 

**What the code does**

Upgrade pravega version to the latest commit pravega/pravega@a621c55

**How to verify it**

`gradlew build` passes

### CODE REVIEW

**Review by claudiofahey — CHANGES_REQUESTED** [review]
I don't think we want to upgrade the master branch to 0.10.0 yet. Per our compat matrix at https://github.com/pravega/spark-connectors#compatibility-matrix, master should contain Pravega 0.9.x. Is there a particular reason to upgrade to 0.10.0 before the release?

**Review by claudiofahey — APPROVED** [review]

**Review by crazyzhou — CHANGES_REQUESTED** [review]
Please update the compatibility matrix together in this PR

**Comment by thekingofcity** [comment]
Compatibility matrix updated.

File: README.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ The [master](https://github.com/pravega/spark-connectors) branch will always hav
 
 | Spark Version | Pravega Version | Java Version To Build Connector | Java Version To Run Connector | Git Branch                                                                        |
 |---------------|-----------------|---------------------------------|-------------------------------|-----------------------------------------------------------------------------------|
-| 3.0           | 0.9             | Java 11                         | Java 8 or 11                  | [master](https://github.com/pravega/spark-connectors)                             |
+| 3.0           | 1.0             | Java 11                         | Java 8 or 11                  | [master](https://github.com/pravega/spark-connectors)                             |
+| 2.4           | 1.0             | Java 11                         | Java 8                        | [r0.10-spark2.4](https://github.com/pravega/spark-connectors/tree/r0.10-spark2.4) |
```
Comment (crazyzhou) [inline README.md:43]: This `Java Version To Build Connector` should be `Java 8`.

FYI @claudiofahey , I will help to create this branch based on `r0.9-spark2.4` after this PR goes into master.

File: gradle.properties [hunk 2]
```diff
@@ -25,8 +25,8 @@ slf4jApiVersion=1.7.25
 sparkVersion=3.0.1
 
 # Version and base tags can be overridden at build time.
-connectorVersion=0.9.0-SNAPSHOT
-pravegaVersion=0.9.0-2725.77b1398-SNAPSHOT
+connectorVersion=0.10.0-SNAPSHOT
+pravegaVersion=0.10.0-1.a621c55-SNAPSHOT
```
Comment (crazyzhou) [inline gradle.properties:29]: We can use this `0.10.0-2869.842cbda-SNAPSHOT` now, the commit number goes back to normal after my fix.

**Review by crazyzhou — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 2 above)
Comment (thekingofcity) [reply gradle.properties]: Updated.

File: README.md (hunk 1 above)
Comment (thekingofcity) [reply README.md]: Updated.

**Review by crazyzhou — APPROVED** [review]
LGTM

**Pull request merged**